css/templates/travel/page-vars.css

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:root {
+  --font-color-primary: var(--black);
+  --font-family-primary: "Source Sans Pro", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", OpenSansEmoji, EmojiSymbols, EmojiSymb, Symbola;
+  --line-height: 1.2;
+  --fixed-navbar-height: calc(86 / 16rem);;
+
+  /* activity type colors. */
+  --type-active-color: #fd68ae;
+  --type-artistic-color: #32dada;
+  --type-drinks-color: #4e93f8;
+  --type-fashion-color: #f1c363;
+  --type-food-color: #aa5efd;
+  --type-music-color: #ec8936;
+  --type-nature-color: #14b52a;
+  --type-nightlife-color: #e83a3a;
+  --type-tours-color: #0ece9e;
+  --type-water-color: #3abee8;
+
+  /* input element */
+  --input-padding-y: var(--button-padding-y);
```

Comment:
do you need these? can't you use the `p*` classes

Reply:
I believe we need them, with BassCSS you can't use the `p*` classes to style the input element because it gets overwritten by BassCSS's `input[type=text]` rule, so this is used to override that.